packages/utils/test/inputValidation.test.ts

```diff
@@ -0,0 +1,52 @@
+import { BaseProps, validateInput } from '../src/inputValidation'
+
+describe('inputValidation', () => {
+  function validateFunction(
+    desc: string,
+    itStr: string,
+    inputs: string[],
+    validator: 'phone' | 'integer' | 'decimal',
+    expected: string,
+    props?: BaseProps
+  ) {
+    describe(desc, () => {
```

Comment:
Typically in a test file there's a root `describe` wrapper with children `it` wrappers:
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/test/parsing.test.ts